### DISCUSSION TITLE
Websockets: Send response to all clients except sender

### DISCUSSION THREAD

**Discussion by ekanna** [tokio-rs/axum#1188] [Q&A] [answered]
In chat example, we broadcast all messages to all clients. But for my use case, i want to send response to all clients except sender how to do it ?

For clarity on my question:
https://stackoverflow.com/questions/10058226/send-response-to-all-clients-except-sender

Thanks

**Comment by davidpdrsn** [comment, marked as answer]
See the chat example in the repo. You'll need to manually keep track of the connected clients.

**Reply by ekanna** [reply]
Thank you. I did some silly mistake. Now it is working fine. I just assigned an id to each connection. Now everything is clear.
